### DISCUSSION TITLE
[Feature] Same Albums and Shared Albums View Design in Web?

### DISCUSSION THREAD

**Discussion by pig-cop** [immich-app/immich#2361] [Feature Request]
### The feature

Is there any plan to make the view design for Shared Albums and Albums to be the same in Web?
At the moment the vertical view with small thumbnails in the Shared Albums has a lot of wasted space and I think it makes sense to be like the Albums page.

![albums](https://user-images.githubusercontent.com/1793444/235450605-de010ddc-2f20-4f10-a3bd-3b31801016a8.png)

![shared_albums](https://user-images.githubusercontent.com/1793444/235450618-74dc14b5-b114-4641-adb7-c63ab161264a.png)

### Platform

- [ ] Server
- [X] Web
- [ ] Mobile

**Comment by andrewdunndev** [comment]
I'd echo that this has been a point of severe confusion for my family. I'd moved over most of our historic (directory tree) albums to immich and have a much easier view (grid) where they can only view it via list on web.

It'd be nice to have consistency of grid on web and mobile.

**Reply by alextran1502** [reply]
How does GPhotos do this on the Shared page with a long list of shared album? I have a short list of album shared on my Google account so I didn't know if they change the view after a certain length. The point here is to replicate the UI in GPhotos for the familiar feeling

**Reply by jrasm91** [reply]
I agree about copying Google photos in general, but albums have always been horrible IMO. I think a grid/tile layout would be great with different options on sorting (date created, oldest/newest photo, recently updated, alphabetical, etc.)

**Reply by pig-cop** [reply]
I absolutely agree about the sorting options, I miss them a lot too. The album sections are in general a bit lacking in options.

**Reply by andrewdunndev** [reply]
Having the option of doing a Grid/List would be __good__. The experience in Google photos is bad once you get to ~40+ albums. In my case I have ~500 albums (every holiday, reunion, vacation, etc.) so it's a "scroll forever" situation. Grid makes it so that at-least you can scroll by ~4 at a time and get a larger cover image.

Ultimately something like #2168 or #2073 would be more __robust__.

**Reply by andrewdunndev** [reply]
I should add... there likely is a "user story" for administrators of immich instances that is similar to mine (with historical family photos on disk)...

Personally I am not looking forward to (or waiting for) #1006 because I would actually _rather_ digest my photos into immich natively. The only __missing feature__ for organizing images into albums for me is the concept of tagging/nesting (e.g. #2168 and #2073).

So this means that I am going to digest the hierarchical structure of my historical family photos (~3TB) into immich album by album. It's going to result in ~500-1000 albums for my family to flip through. All shared by a single user to them, so they will show up in their shared section.
